server/app/models/grid_service_certificate.rb

```diff
@@ -0,0 +1,9 @@
+class GridServiceCertificate
+  include Mongoid::Document
+
+  field :subject, type: String # To identify the cert object
+  field :name, type: String, default: 'SSL_CERTS'
+  field :type, type: String, default: 'env' # For future use where we could expose certs as files
```

Comment:
BTW I dislike this `type` vs `name` model - the `name` would either be an environ name or a filesystem path? Why not just have an `env` field, and then perhaps add a `filepath` field in the future?

Not worth breaking consistency over the existing secret API/model, but just IMO.

```yaml
- certificate: foo.com
  env: SSL_CERTS
```

Reply:
yeah, wanted to keep it fully consistent with secrets model